projects/swimlane/ngx-ui/src/lib/components/nag/nag.component.spec.ts

```diff
@@ -48,12 +48,16 @@ describe('NagComponent', () => {
     expect(component.stateChanged.emit).toHaveBeenCalledTimes(2);
   });
 
-  it('can peek into component if watch is set', () => {
+  // TODO: thius test is flaky
```

Comment:
Are these changes needed after the nag PR gets merged?

Reply:
no...